lib/appium.js

```diff
@@ -172,9 +172,18 @@ class AppiumDriver extends BaseDriver {
     inspectObject(caps);
   }
 
-  async createSession (desiredCaps, reqCaps, capabilities) {
-    if (capabilities) {
-      desiredCaps = processCapabilities(capabilities, null, false);
+  /**
+   * Create a new session
+   * @param {Object} desiredCaps JSONWP formatted desired capabilities
+   * @param {Object} reqCaps Required capabilities
+   * @param {Object} capabilities W3C capabilities
+   */
+  async createSession (desiredCaps={}, reqCaps, capabilities) {
+    if (_.isObject(capabilities)) {
```

Comment:
-> isPlainObject

Reply:
What is the purpose of the check? What would the value be other than an object?